src/Features/Core/Portable/PopulateSwitch/PopulateSwitchExpressionHelpers.cs

```diff
@@ -28,6 +28,16 @@ public static ICollection<ISymbol> GetMissingEnumMembers(ISwitchExpressionOperat
                     return SpecializedCollections.EmptyCollection<ISymbol>();
                 }
             }
+            else if (switchExpressionType?.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T &&
+                        switchExpressionType?.TypeArguments[0].TypeKind == TypeKind.Enum)
+            {
+                var underlyingEnum = switchExpressionType.TypeArguments[0];
+                if (!PopulateSwitchStatementHelpers.TryGetAllEnumMembers(underlyingEnum, enumMembers) ||
+                    !TryRemoveExistingEnumMembers(operation, enumMembers))
+                {
+                    return SpecializedCollections.EmptyCollection<ISymbol>();
+                }
+            }
```

Comment:
same code as above.  instead, can you just do the nullable check first, and exract out the nullable type arg and then unilaterally do the enum chec?

Reply:
I've updated the code to reflect your comments